src/uct/ib/dc/verbs/dc_verbs.h

```diff
@@ -25,6 +25,13 @@
     }
 
 
+#if IBV_EXP_HW_TM
+#  define UCT_DC_VERBS_TM_ENABLE_STR  "TM_ENABLE=n"
```

Comment:
maybe put this #ifdef in dc_verbs.c directly? why need another preprocessor define?